packages/quantic/check-sfdx-project.js

```diff
@@ -0,0 +1,39 @@
+const pathlib = require('path');
+const fs = require('fs');
+
+function quanticIsDependency() {
+  const upperDirectories = __dirname.split(pathlib.sep);
+  const firstTwoUpperDirectories = upperDirectories.slice(
+    upperDirectories.length - 3
+  );
+  return (
+    firstTwoUpperDirectories.join(pathlib.sep) ===
+    pathlib.join('node_modules', '@coveo', 'quantic')
+  );
+}
+
+function getProjectPath(path) {
+  const directories = path.split(pathlib.sep);
+  return directories.slice(0, directories.length - 3).join(pathlib.sep);
+}
+
+function main() {
+  try {
+    if (quanticIsDependency()) {
```

Comment:
This condition is not to check if the directory structure is as expected but it's just to check if the pre/post install scripts are executed because a user installed quantic as a dependency or because of running `npm install` in the quantic repo. 
In this second case no need to execute the pre/post install scripts so we just skip them. I dont know if an `else block` is needed in this case to log something  to the user.